app/views/trainings/search.erb

```diff
@@ -0,0 +1,15 @@
+<h1> Search for your training </h1>
+
+<br>
+<%= form_tag training_search_new_path, method: :get, id: "training-lookup-form" do %>
```

Comment:
default method is `:get` so I think we can omit it here

Reply:
done